app/assets/stylesheets/elements/_label.scss

```diff
@@ -7,14 +7,44 @@ label {
   font-style: italic;
   font-size: 95%;
   font-weight: normal;
+
   &--optional {
     color: $brand-primary;
   }
+
   &--required {
     color: $brand-danger;
   }
 }
 
+.label {
+  color: rgba(0, 0, 0, .9);
+}
+
 .label--minor {
   font-weight: normal;
 }
+
```

Comment:
Ported from the warehouse to provide coloring for the drop-down

<img width="301" alt="Screenshot 2025-02-09 at 11 11 15 AM" src="https://github.com/user-attachments/assets/f4fd25df-3dae-4334-a5ae-6c06a51b8f38" />

Reply:
It's good to be consistent if this is an established pattern. But I'm unclear on how this treatment aligns with accessibility standards.

Reply:
Not an established pattern, other than that the button had the danger color.  This is something we should review with design eventually.  It would be nice to have an accessible (and pretty) way to indicate the danger associated with a given menu item.